src/components/NewTodo/NewTodo.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import classNames from 'classnames';
+
+class NewTodo extends React.Component {
+  state = {
+    selectedUser: 0,
+    inputTodoValue: '',
+    inputTodoError: false,
+    selectUserError: false,
+  }
+
+  onChangeSelectOfUser = ({ target: { value } }) => {
+    if (value !== 0) {
+      this.setState({
+        selectedUser: Number(value),
+        selectUserError: false,
+      });
+    } else {
+      this.setState({
+        selectedUser: Number(value),
+      });
+    }
+  }
+
+  onChangeInputOfTodo = ({ target: { value } }) => {
+    if (value !== '') {
```

Comment:
аналогично с пустой строкой
```suggestion
    if (value) {
```